tests/test_distribution.py

```diff
@@ -0,0 +1,56 @@
+""" Tests for the distribution container """
+import json
+import os
+
+import pytest
+from pytest_container import OciRuntimeBase
+
+from bci_tester.data import DISTRIBUTION_CONTAINER
+from bci_tester.runtime_choice import DOCKER_SELECTED
+
+CONTAINER_IMAGES = [DISTRIBUTION_CONTAINER]
+
+
+def test_registry_service(
+    host, auto_container_per_test, tmp_path, container_runtime: OciRuntimeBase
+):
+    """run registry container with attached volume '/var/lib/docker-registry'"""
+    engine = container_runtime.runner_binary
+    host_port = auto_container_per_test.forwarded_ports[0].host_port
+    content = """
+    FROM alpine:latest
+    CMD ["echo", "container from my local registry"]
+    """
+
+    out = host.check_output(
+        f"curl -sb -H 'Accept: application/json' http://localhost:{host_port}/v2/_catalog"
+    )
+    assert out == '{"repositories":[]}'
```

Comment:
recommendation: use the `json` module here, this test could easily break if they decide to format the json slightly differently

Reply:
Ok, will do. I guess same on L#52

Reply:
Updated